config/src/internal/routing/bgp.rs

```diff
@@ -179,6 +179,101 @@ impl Default for BgpOptions {
     }
 }
 
+#[derive(Clone, Debug)]
+pub enum BmpSource {
```

Comment:
I'd define this in another file (bmp.rs), since the bgp.rs is becoming large and bmp is a clear cut?
(I mean the whole bmp config). In bgp.rs you would just import the BmpOptions.